contrib/storage-hive/core/src/main/java/org/apache/drill/exec/planner/sql/HivePartitionDescriptor.java

```diff
@@ -151,7 +152,7 @@ protected void createPartitionSublists() {
   }
 
   @Override
-  public TableScan createTableScan(List<PartitionLocation> newPartitions) throws Exception {
+  public TableScan createTableScan(List<PartitionLocation> newPartitions, String cacheFileRoot) throws Exception {
```

Comment:
From the comment, cacheFileRoot could be null. But is it possible to not adding this parameter to HivePartitionDescriptor? After all, cacheFileRoot will not be applied to hive partition.